scripts/publish_gh_release_notes.py

```diff
@@ -71,14 +72,19 @@ def main(argv):
         print("GH_RELEASE_NOTES_TOKEN not set", file=sys.stderr)
         return 1
 
+    slug = os.environ.get("TRAVIS_REPO_SLUG")
+    if not slug:
+        print("TRAVIS_REPO_SLUG not set", file=sys.stderr)
+        return 1
```

Comment:
Just use `slug = os.environ["TRAVIS_REPO_SLUG"]` maybe to get a traceback also?  (i.e. more info)
But it's OK like that also.

Reply:
I think the message is pretty clear TBH 😁

Reply:
Sure.
I've just meant that just asserting it to be there is enough/shorter, and provides validation (all in a single statement).